### ISSUE TITLE
Mobile Wallet - Android Version Dependency of V8(+)

### ISSUE THREAD

**Issue by MrCryptoT** [mwcproject/mwc-qt-wallet#818]
**Description:**
The Mobile wallet has an Android Version dependency for V8 **and** higher.
This is quite likely a Limitation and not a bug in the classical sense. Suggest "wontfix" and "documentation" as labels, can be closed afterwards imho. I already informed @sarawut11 via discord, and would like this Issue "archived" (closed) just to find it again if this error comes up again. I myself encountered this Bug in my "life environment" due to lazy updating so I think this may come up once or twice when releasing originally to endusers. 

I did not reproduce myself, as I tested this on my Smasung S7 and not an emulator, but am confident in this Limitation as I retested the wallet on "the way up" and it worked again in V8.0.0.

CC FYI: @jbyrer @vek-kal @vinayaga07

**Reproducing:** 
Install an Emulator Android Version of 7.x.x and open the wallet to click "new wallet" or "restore wallet". 
The wallet crashes with below error. (Symptom)

**Verify Limitation:** 
Upgrade Emulator or Phone version to android 8.0.0 and retry, wallet will be able to create a new wallet/restore an existing wallet now. 

**Symptom:** 
The Wallet installs and starts normally, but crashes with the following error as soon as you click anything: 

> Wallet got critical error: 
> mwc713 executable is not found. 
> Expected location at /data/app/mw.mwc.wallet-1/lib/arm/64/mwc713.so 
> 
> press ok to exit

Seems related to how Android searches Libraries of APK's in older Versions as the path seems to change in newer versions

Thanks for everyones time and as always, have a good ☕

**Comment by bayk** [comment]
By design it suppose to work from Android versions 5.0.   Will check. That might be QT issue.

**Comment by bayk** [comment]
The problem is bigger than just file location. I tested with Android 5.1.  By some reasons our binary files are not extraction into lib directory. There are 3 files, none of them were there.  Apk file has them.  If I copy those files manually, mwc713 failing with "Segmentation fault" error. 
The problem is likely dependency for the Android OS at Android emulator.  Such problem will be hard to fix. We are building with latest Android NDK, so probably it is a problem. Before with prev NDK I tested and it worked fine. Because rust is expected latest NDK as well, we can't downgrade.

x86 platform is really for debugging only and we don't want spend much time on that. That is why making this ticker low priority.
